### PR TITLE
Update html link workout

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Remove duplicate codepen link

Remove the * `#` from the Practice question to make the question easier and leave the * `#` as the Revision question.

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/web/html/links/anchor-links.md?ref=24cb8224c20c27c0b774791d62ea1bf44c1895c5

File: web/html/links/anchor-links.md [hunk 1]
```diff
@@ -57,20 +55,20 @@ This is a
...
  contact section
 </tag> of the page.
 ```
 tag = ???
 
 * `href`
-* `#`
 * `a`
 * `style`
 * `p`
```
Comment (sourabhv) [inline web/html/links/anchor-links.md]: How about we change the question to:

    ```html
    Link to the
    <??? ???="#contact.html">
     contact section
    <???> of the page.
    ```

    * `a`
    * `href`
    * `/a`
    * `style`
    * `p`

File: web/html/links/link-relative-paths.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ Match the image code with the path descriptions:
 `<img src="../../logo.svg">`: ???
 `<img src="../images/logo.svg">`: ???
 
-Select the reason relative links are not considered the best for images and files within a website.
+Select the reason relative links are NOT considered the best for images and files within a website.
```
Comment (sourabhv) [inline web/html/links/link-relative-paths.md]:
```suggestion
Select the reason relative links are **not** considered the best for images and files within a website.
```

File: web/html/links/anchor-links.md (hunk 1 above)
Comment (nem035) [reply web/html/links/anchor-links.md]: @sourabhv having only `/a` as the closing tag in the answers makes it an obvious answer 🤷‍♂ 

Maybe something like?

    ```html
    Link to the
    <??? ???="#contact.html">
     contact section
    <???> of the page.
    ```

    * `a`
    * `href`
    * `/a`
    * `url`
    * `/p`

Comment (sourabhv) [reply web/html/links/anchor-links.md]: How about we change the question to:

    ```html
    Link to the
    <??? ???="#contact.html">
     contact section
    <???> of the page.
    ```

    * `a`
    * `href`
    * `/a`
    * `style`
    * `link`
    * `/link`
    * `url`
    * `p`
    * `/p`

Comment (Stefan-Stojanovic) [reply web/html/links/anchor-links.md]: That looks better, I've added it

**Pull request merged**